python/graphstorm/utils.py

```diff
@@ -85,8 +86,14 @@ def barrier():
 def use_wholegraph(part_config):
```

Comment:
Where does use_wholegraph be called?
It seems use_wholegraph will not be called and USE_WHOLEGRAPH will always be false.

Reply:
from `gs.initialize()` which is always called